src/components/GraphViewer/GraphStructure.jsx

```diff
@@ -0,0 +1,28 @@
+const data = {
```

Comment:
@jrmartin question here, can't we derive the data for the graph from the predicates that we get from the endpoint? Thx

Reply:
@ddelpiano Yes, that's the idea. However, the endpoint returning the predicates is not yet in devel branch. The PR #26 with predicates is now open, and once merge to devel I can bring changes here to replace it with endpoint.